src/main/java/teammates/ui/webapi/output/CourseData.java

```diff
@@ -0,0 +1,43 @@
+package teammates.ui.webapi.output;
+
+import teammates.common.datatransfer.attributes.CourseAttributes;
+
+/**
+ * API output for a course.
+ */
+public class CourseData extends ApiOutput {
```

Comment:
@ChooJeremy  I saw you have implemented your own version of this, have you two discussed about this